ConcernsCaseWork/ConcernsCaseWork/Services/Nti/NtiModelService.cs

```diff
@@ -16,10 +17,52 @@ public NtiModelService(INtiCachedService ntiCachedService)
 			_ntiCachedService = ntiCachedService;
 		}
 
+		public async Task<NtiModel> CreateNtiAsync(NtiModel ntiModel)
+		{
+			var created = await _ntiCachedService.CreateNtiAsync(NtiMappers.ToDBModel(ntiModel));
+			return NtiMappers.ToServiceModel(created);
+		}
+
+		public async Task<NtiModel> GetNtiAsync(string continuationId)
+		{
+			if (string.IsNullOrWhiteSpace(continuationId))
```

Comment:
Are we considering guard clauses going forward, or is this still being decided?

Reply:
This story was already being implemented when the discussion started. Will start using guard clauses going forward.